libraries/eosiolib/tester/wasi_polyfill/__wasi_fd_fdstat_get.cpp

```diff
@@ -0,0 +1,11 @@
+#include <wasi/api.h>
+
+extern "C" __wasi_errno_t __wasi_fd_fdstat_get(__wasi_fd_t fd, __wasi_fdstat_t* stat)
+    __attribute__((__import_module__("wasi_snapshot_preview1"), __import_name__("fd_fdstat_get")))
+{
+   stat->fs_filetype = __WASI_FILETYPE_CHARACTER_DEVICE;
+   stat->fs_flags = __WASI_FDFLAGS_APPEND;
+   stat->fs_rights_base = __WASI_RIGHTS_FD_READ | __WASI_RIGHTS_FD_WRITE;
```

Comment:
Are there and readable file descriptors?

Reply:
Before: ish. Even though stdin wasn't really hooked up, this allowed musl to not break during initialization. Having extra rights on stdin, stdout, and stderr didn't hurt it.

Now: replaced with native-side implementation which deals with files in addition to stdin, stdout, and stderr